ui/lib/core/addon/components/masked-input.hbs

```diff
@@ -26,7 +31,7 @@
   {{/if}}
   {{#if @allowCopy}}
     <CopyButton
-      @clipboardText={{@value}}
+      @clipboardText={{if (eq @value "") " " @value}}
```

Comment:
I ended up hardcoding in the space `" "` for when there is no value `""` because otherwise it wasn't giving the desired behavior. 

First I thought a good solution was just adding an error message identical to the success message, that way a user doesn't realize it is an empty value until they either paste what is copied or they reveal the value, as we discussed on slack. The problem is that this doesn't clear the clipboard. So if they user has something copied before and they click copy, they would get the error message (which looks just like the success message) but their clipboard would remain the same

I decided to just copy a space instead so that the clipboard is actually changing. Otherwise I thought it would be confusing for the user. I couldn't find a way to clear the clipboard with Ember's `<CopyButton>` but if you know a way please let me know. Or I can change it to show an error identical to the success message and the clipboard will just remain the same. Let me know what you think works best!

Reply:
I think as a user I wouldn't expect the clipboard to update if there was a problem copying (I'd expect the last copied item to be there). I didn't realize that the `<CopyButton>` took an `@error` argument - so I think it'd be useful and an overall UX improvement if we surfaced to the user if there was a problem copying. 

I know in slack we had discussed showing a success - but that was before I looked at the component and realized we could have different behavior if there was an error. So I think surfacing a `"danger"` flash message (instead of success) saying something like "Error copying data" would be the best approach here 🤔

Reply:
Yeah, that makes sense! Will do, thanks!